src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -396,6 +405,7 @@ public Task RenderPartialAsync([NotNull] string partialViewName, object model, V
             var templateBuilder = new TemplateBuilder(_viewEngine,
                                                       ViewContext,
                                                       ViewData,
+                                                      TempData,
```

Comment:
Why does this need to be passed through separately when it's already part of the ViewContext?